src/App.jsx

```diff
@@ -82,9 +83,8 @@ function App() {
                 </PrivateRoute>
                 <Route path="/missions/:id" component={MissionDetails} />
                 <Route path="/missions" component={Missions} />
-                <PrivateRoute path="/user">
-                  <UserProfile />
-                </PrivateRoute>
+                <Route path="/user" component={UserProfile} />
```

Comment:
Might be a bigger discussion, but should the route be something like /user/:userId ? @mat10tng @utunga ?

Reply:
I am going to add another route soon, so that route is going to be /user/profile later

Reply:
awesome! wanted to verify that.